src/components/Moviepage/MoviePage.jsx

```diff
@@ -1,5 +1,64 @@
-import React from "react";
+import axios from "axios";
+import React, { Suspense, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import Spinner from "../Spinner/Spinner";
 
-export default function MoviePage({ movieInfo }) {
-  return <div>{movieInfo.id}</div>;
+const api_key = process.env.REACT_APP_API_KEY;
+
+export default function MoviePage() {
+  const [movieInfo, setMovieInfo] = useState([]);
+  const params = useParams();
+
+  useEffect(() => {
+    fetchMovieData(params.id);
+  }, []);
+
+  async function fetchMovieData(id) {
+    try {
+      const response = await axios.get(
+        `https://api.themoviedb.org/3/movie/${id}`,
+        {
+          params: {
+            api_key: api_key,
+          },
+        }
+      );
+      setMovieInfo(response.data);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  return (
+    <Suspense fallback={<Spinner />}>
+      <article data-testid="movieID">
```

Comment:
You should use the same naming model for all data-testids in your project. You can use camelCase or kebab-case.

Reply:
fixed